scripts/update-client.sh

```diff
@@ -22,7 +22,7 @@ set -o nounset
 set -o pipefail
 
 # The openapi-generator version used by this client
-export OPENAPI_GENERATOR_COMMIT="v4.3.0"
+export OPENAPI_GENERATOR_COMMIT="v7.1.0"
```

Comment:
I think we need to first regenerate this client and review the diff. In particular we want to make sure the breaking changes are captured, and tests continue to pass.